mobile/src/components/uam/components/uam-group-creation/components/permissions-table/permissions-table.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+import { useWindowDimensions } from 'react-native';
+
+import { PermissionsGetAllItemResponseDto } from '~/common/types/types';
+import { Table, View } from '~/components/common/common';
+import {
+  getPermissionsColumns,
+  getPermissionsRows,
+} from '~/components/uam/components/uam-group-creation/helpers/helpers';
+import { useAppForm } from '~/hooks/hooks';
+
+type Props = {
+  permissions: {
+    items: PermissionsGetAllItemResponseDto[];
+  };
```

Comment:
You should pass there only permissions
```suggestion
  permissions: PermissionsGetAllItemResponseDto[]
```